index.js

```diff
@@ -50,7 +62,25 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies) {
```

Comment:
```suggestion
function checkIfAnyMovieHasRating(movies, rating = "G") {
```
`rating = "G"` is the syntax for setting a default parameter.